src/components/tx/TxSimulation/SimulationResult.tsx

```diff
@@ -15,6 +15,14 @@ type SimulationResultProps = {
   onClose: () => void
 }
 
+const getCallTraceErrors = (simulation?: TenderlySimulation) => {
+  if (!simulation) {
+    return []
+  }
+
+  return simulation.transaction.call_trace.filter((call) => call.error)
```

Comment:
Why do we go for call errors instead of the `ExecutionFailure` event as written in the comment?

We could iterate over `simulation.transaction.logs` and filter by `log.name === 'ExecutionFailure' && log.raw.address === safeAddress` emitted from `safeAddress`.

Reply:
For future reference: after discussing this, differing Safe versions emit different events (<1.1.0 emits `ExecutionFailed`) so the logs aren't consistent.

As such, we've adjusted the code to only extract the call traces if the transaction simulation "succeeds".